lib/groupBy/index.js

```diff
@@ -0,0 +1,12 @@
+import _curry2 from '../_internal/_curry2'
+
+export default _curry2(function groupBy(func, list) {
+  var obj = {}
+  for (var i = 0; i < list.length; i++) {
+    var key = func(list[i])
+    obj.hasOwnProperty(key)
+      ? obj[key].push(list[i])
+      : obj[key] = [list[i]]
```

Comment:
maybe replace with `obj[key] = (obj[key] || []).concat(list[i])`? Or is it slower?

Reply:
tested `concat`, it's slower, because it creates new array every time and redefining object[key] every time. Test crushes with 1e3/1e4/1e4 arrays

Reply:
maybe replace `obj.hasOwnProperty(key)` with `obj[key]`?

Reply:
`obj[key]` will search through prototype chain. I think we do not want such behavior, but It's opened question